### ISSUE TITLE
Admin page is available even without ADMIN_TOKEN

### ISSUE THREAD

**Issue by poblabs** [dani-garcia/bitwarden_rs#515]
I admit I'm a bit new to bitwarden_rs, so I'm still trying to figure it out. I'm running this in a docker container on my Synology and it's working great. 

I **do not** have the `ADMIN_TOKEN`, nor `DISABLE_ADMIN_TOKEN` environment variables specified, yet `/admin` is available to me without authentication. 

Anything I'm missing? I would assume that page would be disabled if I wasn't explicitly defining something with those environment variables. 

Here's the version from the header of the admin page: `Version: 1.9.1-3fb63bbe`

**Comment by mprasil** [comment]
which docker image are you using - based on the version hash, it seems to be the latest version of `bitwardenrs/server`, is that right? Can you describe steps to reproduce it?

Have you at any stage set the admin token via the admin interface? These settings are saved into a configuration file, so your ADMIN_TOKEN can be still set there even if you removed the environment variable.

**Comment by poblabs** [comment]
> which docker image are you using - based on the version hash, it seems to be the latest version of bitwardenrs/server, is that right?

That's right. I downloaded it last night. The version hash looks like your latest server commit hash. 

> Can you describe steps to reproduce it?

I installed the image, started a container, and was able to browse to http://<IP>/admin without any challenge. No variables are defined which would enable the admin page (according to the wiki docs)

> Have you at any stage set the admin token via the admin interface?

Yes, I set it once to test it out. Ran into some issues, so I stopped the container, removed the variable, restarted the container. So this makes sense - however if the variable is saved in a cache file, I question why I'm not being challenged?

Note: I also did set `DISABLE_ADMIN_TOKEN` as well at one point - but I have since removed it. Where I feel I should be right now is with a disabled admin page.

Where are these variables cached? I'll be curious to try and find/remove them and see how it goes.
